boards/toradex/verdin_imx8mm/doc/index.rst

```diff
@@ -0,0 +1,268 @@
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | GPIO output                         |
+|           |            | GPIO input                          |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/arm/verdin_imx8mm_m4/verdin_imx8mm_m4_defconfig`.
+
+It is recommended to disable peripherals used by the M4 core on the Linux host.
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+UART:
+
+Zephyr is configured to use the UART4 by default, which is connected to the FTDI
+USB converter on most Toradex carrier boards.
+
+This is also the UART connected to WiFi/BT chip in modules that have the WiFi/BT
+chip. Therefore, if UART4 is used, WiFI/BT will not work properly.
+
+If the WiFi/BT is needed, then another UART should be used for Zephyr (UART2 for
+example). You can change the UART by changing the ``zephyr,console`` and
+``zephyr,shell-uart`` in the :zephyr_file:`boards/arm/verdin_imx8mm_m4.dts` file.
+
++---------------+-----------------+---------------------------+
+| Board Name    | SoC Name        | Usage                     |
++===============+=================+===========================+
+| UART_1        | UART2           | General purpose UART      |
++---------------+-----------------+---------------------------+
+| UART_2        | UART3           | General purpose UART      |
++---------------+-----------------+---------------------------+
+| UART_3        | UART1           | Cortex-A53 debug UART     |
++---------------+-----------------+---------------------------+
+| UART_4        | UART4           | Cortex-M4 debug UART      |
++---------------+-----------------+---------------------------+
+
+GPIO:
+
+All the GPIO banks are enabled in the :zephyr_file:`dts/arm/nxp/nxp_imx8m_m4.dtsi`.
+
+LED:
+
+There is no LED in the module itself, this is dependent on the carrier board that
+is being used with the module. The device tree is configured to use the GPIO3_IO1,
+which can be connected to the LED of the Verdin Development Board or changed in the
+:zephyr_file:`boards/arm/verdin_imx8mm_m4.dts` if needed.
```

Comment:
File path out of date

Reply:
This has been addressed in the latest push.